watertap/unit_models/tests/test_anaerobic_digestor.py

```diff
@@ -163,269 +110,104 @@ def adm(self):
-    def test_conservation(self, adm):
-        assert (
-            abs(
-                value(
-                    adm.fs.unit.inlet.flow_vol[0] * adm.fs.props.dens_mass
-                    - adm.fs.unit.liquid_outlet.flow_vol[0] * adm.fs.props.dens_mass
-                    - adm.fs.unit.vapor_outlet.flow_vol[0] * adm.fs.props_vap.dens_mass
-                )
-            )
-            <= 1e-6
-        )
-
-        assert pytest.approx(-13.5835, abs=1e-3) == value(
-            adm.fs.unit.liquid_phase.enthalpy_transfer[0]
-        )
-        assert (
-            abs(
-                value(
-                    (
-                        adm.fs.unit.inlet.flow_vol[0]
-                        * adm.fs.props.dens_mass
-                        * adm.fs.props.cp_mass
-                        * (
-                            adm.fs.unit.inlet.temperature[0]
-                            - adm.fs.props.temperature_ref
-                        )
-                    )
-                    - (
-                        adm.fs.unit.liquid_outlet.flow_vol[0]
-                        * adm.fs.props.dens_mass
-                        * adm.fs.props.cp_mass
-                        * (
-                            adm.fs.unit.liquid_outlet.temperature[0]
-                            - adm.fs.props.temperature_ref
-                        )
-                    )
-                    + adm.fs.unit.liquid_phase.enthalpy_transfer[0]
-                )
-            )
-            <= 1e-2
-        )
-
-    @pytest.mark.solver
-    @pytest.mark.skipif(solver is None, reason="Solver not available")
-    @pytest.mark.component
-    def test_costing(self, adm):
-        m = adm
-
+        # Add unit model costing
         m.fs.costing = WaterTAPCosting()
 
         m.fs.unit.costing = UnitModelCostingBlock(flowsheet_costing_block=m.fs.costing)
         m.fs.costing.cost_process()
-        m.fs.costing.add_LCOW(m.fs.unit.liquid_phase.properties_out[0].flow_vol)
-        solver = get_solver()
-        results = solver.solve(m)
-
-        assert_optimal_termination(results)
-
-        # Check solutions
-        assert pytest.approx(2.0 * 1083290.8, rel=1e-5) == value(
-            m.fs.unit.costing.capital_cost
-        )
-        assert pytest.approx(5.2754, rel=1e-5) == value(m.fs.costing.LCOW)
-
-    @pytest.mark.unit
-    def test_get_performance_contents(self, adm):
-        perf_dict = adm.fs.unit._get_performance_contents()
 
-        assert perf_dict == {
-            "vars": {
-                "Volume": adm.fs.unit.volume_AD[0],
-                "Heat Duty": adm.fs.unit.heat_duty[0],
-            }
-        }
-
-    @pytest.mark.unit
-    def test_report(self, adm):
-        adm.fs.unit.report()
+        # Set scaling factors for badly scaled variables
+        iscale.set_scaling_factor(
+            m.fs.unit.liquid_phase.mass_transfer_term[0, "Liq", "S_h2"], 1e7
+        )
+        iscale.set_scaling_factor(m.fs.unit.costing.capital_cost, 1e-6)
```

Comment:
Didn't verify, but do we apply these same scaling factors on the BSM2 flowsheet? Just curious

Reply:
I also wonder whether we should add this scaling to the unit itself under its `calculate_scaling_factors ` method. I know that isn't the focus of the PR, but if you can probe into this and address as needed in a separate issue/follow-on PR, that would be great. For example, would adding the scale factor for `mass_transfer_term` to the unit break other tests (i.e., BSM2)?

Reply:
These scaling factors aren't applied in the BSM2 flowsheet.

I'm not sure that we should set this scaling factor on the unit model since it's only applying the scaling to a single component.